packages/schematics/angular/utility/find-module_spec.ts

```diff
@@ -111,12 +111,20 @@ describe('find-module', () => {
       expect(modPath).toEqual('/projects/my-proj/src/app.module.ts' as Path);
     });
 
-    it('should find a module if nameFormatter is provided', () => {
```

Comment:
this test was actually not testing anything, as `nameFormatter ` was only used when `name` is provided